iexec/src/main/scala/gridscale/iexec/package.scala

```diff
@@ -0,0 +1,83 @@
+package gridscale
+
+import effectaside._
+import gridscale.cluster.BatchScheduler.BatchJob
+import gridscale.cluster.{ BatchScheduler, HeadNode }
+import gridscale.tools._
+import monocle.macros._
+import squants._
+import squants.information._
+
+package object iexec {
+
+  case class IEXECJobDescription(
+    workDirectory: String,
+    IexecFilesPath: String,
+    dappAddress: String,
+    arguments: String,
+    dappCost: Int)
+
+  object impl {
+
+    def toIexec(description: IEXECJobDescription) =
+      s"""
+         |#!/bin/bash
+         |export PATH="${description.IexecFilesPath}:$${PATH}"
+         |cd ${description.workDirectory}
+         |iexec account login
```

Comment:
iExec maintains a session based on your wallet address => you shouldn't have to login everytime, you can call iexec account show first to see if you're logged in